Rakefile

```
@@ -20,3 +20,14 @@ require 'hammer_cli_foreman_ansible/version'
 require 'hammer_cli_foreman_ansible/i18n'
 require 'hammer_cli/i18n/find_task'
 HammerCLI::I18n::FindTask.define(HammerCLIForemanAnsible::I18n::LocaleDomain.new, HammerCLIForemanAnsible.version.to_s)
+
+begin
+  require 'rubocop/rake_task'
+  RuboCop::RakeTask.new
+rescue StandardError => _e
+  puts 'Rubocop not loaded.'
+end
+
+task :default do
+  Rake::Task['rubocop'].execute
+end
```

You can implement it as a prerequisite
```suggestion
task :default => :rubocop
```

But keep in mind that it will fail if rubocop isn't loaded.

It's also a common pattern to run the test suite as well. So what I would recommend is

```ruby
default_tasks = [:test]

begin
  require 'rubocop/rake_task'
rescue StandardError => _e
  # rubocop not loaded
else
  RuboCop::RakeTask.new
  default_tasks.prepend([:rubocop])
end

task :default => default_tasks
```

---

Thanks for the improvement!

---

> ```ruby
> rescue StandardError => _e
> ```

And I just see this. You can drop `=> _e` as well.